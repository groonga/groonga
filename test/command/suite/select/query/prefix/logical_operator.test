table_create Users TABLE_NO_KEY
column_create Users name COLUMN_SCALAR ShortText

load --table Users
[
["name"],
["Alice"]
]

select Users \
  --match_columns name \
  --query "Bo* OR li"
