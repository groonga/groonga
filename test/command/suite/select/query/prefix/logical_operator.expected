table_create Users TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Users name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Users
[
["name"],
["Alice"]
]
[[0,0.0,0.0],1]
select Users   --match_columns name   --query "Bo* OR li"
[[0,0.0,0.0],[[[1],[["_id","UInt32"],["name","ShortText"]],[1,"Alice"]]]]
